Contact-us/contact-us.html

```diff
@@ -0,0 +1,143 @@
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="collapsibleNavId">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="index.html">Home page</a>
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="#">About us</a>
+                    </li>
+                    <li class="nav-item dropdown active" style="margin-left: 50px;">
+                        <a class="nav-link dropdown-toggle" href="#" id="dropdownId" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Available furniture</a>
+                        <div class="dropdown-menu" aria-labelledby="dropdownId">
+                            <a class="dropdown-item" href="#">Office chairs</a>
+                            <a class="dropdown-item" href="#">Dinner chairs</a>
+                            <a class="dropdown-item" href="#">Rocking chairs</a>
+                            <a class="dropdown-item" href="#">Glider chairs</a>
+                            <a class="dropdown-item" href="#">Sofa</a>
+                        </div>
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="#">News</a>
+                    </li>
+                    <li class="nav-item active" style="margin-left: 50px;">
+                        <a class="nav-link" href="Contact-us/contact-us.html">Contact us</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+    <!--Showrooms-->
+    <div class="container">
+        <div class="row pt-md-3 pt-3">
+            <div class="showroom col-md-6">
+                <h6 style="vertical-align: middle;">SHOWROOM NGUYEN THI MINH KHAI</h6>
+                <p>
+                    Address: 264A - 264B - 264C Nguyen Thi Minh Khai, Ward 6, District 3, Ho Chi Minh City, Vietnam. Hotline: 18006867 - 02873016867 Open time: 08h00 - 21h30
+                </p>
+            </div>
+            <div class="mapouter col-md-6">
+                <div class="gmap_canvas"><iframe width="100%" height="300px" id="gmap_canvas" src="https://maps.google.com/maps?ll=10.773922,106.6894049&q=264A Nguyễn Thị Minh Khai&t=&z=14&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                    <a href="https://techwithlove.com/"></a>
+                </div>
+            </div>
+        </div>
+
+        <div class="row pt-md-3 pt-3">
+            <div class="showroom col-md-6">
```

Comment:
default col ?

Reply:
added `col-12` for each `.showroom` and `.mapouter` as default col